engineering/ember.md

```diff
@@ -116,8 +135,35 @@ fullName: computed('firstName', 'lastName', {
 
 ```
 
+## Glimmer Components
+
+### Component Args
+
+- constructor with setting local props, args with objects, chain in template or getter in JS?
+
+### Tracked properties
+
+- Concerns for mixed use of computed props passed down from existing controllers
+
+### Computed properties
+
+- Using computed properties instead of getter and setter
+
+
+## Tailwind CSS
+
+- Using important when tailwind css is overwritten or adding to scss?
+
+
+
 ## Templates
 
+### Actions
```

Comment:
Perhaps Need to document use of `fn` and `@action` for replacing old computed props